hazelcast/src/main/java/com/hazelcast/internal/cluster/impl/TcpIpJoiner.java

```diff
@@ -429,8 +460,7 @@ public void searchForOtherClusters() {
             logger.severe(e);
             return;
         }
-        possibleAddresses.remove(node.getThisAddress());
-        possibleAddresses.removeAll(node.getClusterService().getMemberAddresses());
+        possibleAddresses.removeAll(node.getLocalAddressRegistry().getLocalAddresses().getAllAddresses());
```

Comment:
We will send the `SplitBrainMergeValidationOp` to the existing members in the cluster (as we only remove local addresses).
Is it intentional?

Reply:
not intentional, that's my mistake. I'll fix it

Reply:
Thanks, seems ok now.